clipboard/src/web.ts

```diff
@@ -0,0 +1,100 @@
+import { WebPlugin } from '@capacitor/core';
+import {
+  ClipboardPlugin,
+  ClipboardWrite,
+  ClipboardReadResult,
+} from './definitions';
+
+declare var navigator: any;
+declare var ClipboardItem: any;
+
+export class ClipboardWeb extends WebPlugin implements ClipboardPlugin {
+  constructor() {
+    super({
+      name: 'Clipboard',
+      platforms: ['web'],
+    });
+  }
+
+  async write(options: ClipboardWrite): Promise<void> {
+    if (!navigator.clipboard) {
+      return Promise.reject('Clipboard API not available in this browser');
+    }
+
+    if (options.string !== undefined || options.url) {
+      if (!navigator.clipboard.writeText) {
+        return Promise.reject(
+          'Writting to clipboard not supported in this browser',
+        );
+      }
+      await navigator.clipboard.writeText(
+        options.string !== undefined ? options.string : options.url,
+      );
+    } else if (options.image) {
+      if (!navigator.clipboard.write) {
```

Comment:
I think we need to do more here. [`navigator.clipboard.write`](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/write) seems to have more browser adoption than [`ClipboardItem`](https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem).

```suggestion
      if (!navigator.clipboard.write || !ClipboardItem) {
```